lib/rubocop/cop/rails/skips_model_validations.rb

```diff
@@ -77,12 +83,27 @@ def allowed_method?(node)
             !node.arguments?
         end
 
-        def blacklist
-          cop_config['Blacklist'] || []
+        def forbidden_methods
+          obsolete_result = cop_config['Blacklist']
```

Comment:
For compatibility, an empty array will have to be returned if nil.

```suggestion
          obsolete_result = cop_config['Blacklist'] || []
```

Reply:
If you do this, the conditional below will always return 'true', right? We want this to be nil and then advance to looking for `ForbiddenMethods`.

Reply:
Ah, sure. My above feedback is wrong, and this code is good.